index.js

```diff
@@ -172,7 +163,8 @@ module.exports = class Hyperdrive extends ReadyResource {
       await blobsCore.ready()
 
       this.blobs = new Hyperblobs(blobsCore)
-      this.db.metadata.contentFeed = this.blobs.core.key
+      const db = this.db.tree || this.db
+      db.metadata.contentFeed = this.blobs.core.key
```

Comment:
Same here about `.tree`, I think it's ok

Reply:
i think the batch needs a getter for the metadata or a static hyperbee method that returns the db from either a batch or bee

Reply:
Setter*?

I think we're already in the path of making the Batch as similar as Hyperbee so a setter for the metadata makes sense. Same about adding `getHeader` method to Batch

So do I add that or we just leverage the `getBee` helper in Hyperdrive now?